hub/powertoys/grouppolicy.md

```diff
@@ -1,17 +1,21 @@
 ---
 title: PowerToys Group Policy
 description: Group policy documentation for PowerToys
-ms.date: 10/13/2023
+ms.date: 11/05/2023
 ms.topic: article
 no-loc: [PowerToys, Windows, Group Policy, Win]
 ---
 
 # Group Policies
 
-Since version 0.64, PowerToys is released on GitHub with [Administrative Templates that allows you to configure PowerToys using Group Policies](/previous-versions/windows/desktop/policy/group-policy-objects). You can check these releases on <https://github.com/microsoft/PowerToys/releases>.
+Since version 0.64, PowerToys is released on GitHub with Administrative Templates that allows you to configure PowerToys using Group Policies.
 
 ## How to install
 
+### Download
+
+You can find the latest administrative templates (ADMX files) in the assets section of our newest PowerToys release on <https://github.com/microsoft/PowerToys/releases>. The file is named `GroupPolicyObjectsFiles-<Version>.zip`.
```

Comment:
@crutkas, @jaimecbernardo 
I am worry that the file name gets out of sync/wrong if we can't automate release asset creation. And imo we should standardize the name to `AdministrativeTemplates-<Version>.zip`.

xref:

Reply:
i'm fine for wathever, this is pretty straight forward tweak and like i said, work i want to get done for releasing.